packages/connect/src/protocol-grpc-web/handler-factory.ts

```diff
@@ -175,7 +175,7 @@ function createHandler<I extends Message<I>, O extends Message<O>>(
     );
     return {
       ...uResponseOk,
-      body: outputIt,
```

Comment:
suggestion: I'm not sure if this suggestion would be strange for this codebase but it'd be nice to leave a comment here (or somewhere?  maybe the untilFirst JSDoc itself?) explaining why this is necessary and maybe some detail about how node's behavior differs from something like cloudflare workers.